### PR TITLE
feat: add watcher fixtures

### PR DESCRIPTION
**Pull request by Zidious**
Please see referenced ticket

### CODE REVIEW

**Review by michael-siek — APPROVED** [review]

File: fixtures/watcher/watcher.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        <script>
+          function updateColour() {
+            const colorBlock = document.getElementById("color-block");
+            colorBlock.style.backgroundColor = "gray";
+            colorBlock.style.display = "block";
+
+            document.title = "Watcher - Test Case 1 - Button Click";
+          }
+        </script>
+      </div>
```
Comment (Zidious) [inline fixtures/watcher/watcher.html:36]: Decided to put `script` and `style` tags within each `div` "test block". Can merge them all necessary, this reads easier but no strong opinion here.

**Comment by straker** [comment]
Is this the right place for this test fixture? We've been using this strictly for all the apis to have tests, watcher is outside that scope. Is there a reason this doesn't belong in the watcher repo?

**Review by straker — DISMISSED** [review]
I don't think we should use this repo as a place to gather any fixtures we need shared across multiple repos. We should limit the scope of this repo to test cases and fixtures that are needed by axe-core APIs

**Comment by Zidious** [comment]
> I don't think we should use this repo as a place to gather any fixtures we need shared across multiple repos. We should limit the scope of this repo to test cases and fixtures that are needed by axe-core APIs

Clarification for readers: discussed during standup - watcher related fixtures will have a home here under `/fixtures/watcher`

**Review by michael-siek — APPROVED** [review]

**Pull request merged**